src/watertap_contrib/seto/costing/units/chemical_softening_0D.py

```diff
@@ -0,0 +1,833 @@
+        domain=NonNegativeReals,
+        units=blk.costing_package.base_currency,
+    )
+
+    blk.lime_feed_system_capital_cost = Var(
+        initialize=1e6,
+        doc="Capital cost of lime feed system",
+        domain=NonNegativeReals,
+        units=blk.costing_package.base_currency,
+    )
+
+    blk.admin_capital_cost = Var(
+        initialize=1e6,
+        doc="Capital cost of administrative, laboratory, and maintenance building",
+        domain=NonNegativeReals,
+        units=blk.costing_package.base_currency,
+    )
+    # endregion
+
+    # region Operating cost components
+
+    blk.mix_tank_op_cost = Var(
+        initialize=1e4,
+        doc="Operational cost of mixing tank - energy consumption and labor",
+        domain=NonNegativeReals,
+        units=blk.costing_package.base_currency,
+    )
+
+    blk.floc_tank_op_cost = Var(
+        initialize=1e4,
+        doc="Operational cost of flocculation tank - energy consumption and labor",
+        domain=NonNegativeReals,
+        units=blk.costing_package.base_currency,
+    )
+
+    blk.sed_basin_op_cost = Var(
+        initialize=1e4,
+        doc="Operational cost of sedimentation basin - energy consumption, labor, and maintenance",
+        domain=NonNegativeReals,
+        units=blk.costing_package.base_currency,
+    )
+
+    blk.recarb_basin_op_cost = Var(
+        initialize=1e4,
+        doc="Operationg cost of recarbonation basin - energy consumption, labor, and maintenance",
+        domain=NonNegativeReals,
+        units=blk.costing_package.base_currency,
+    )
+
+    blk.lime_feed_op_cost = Var(
+        initialize=1e4,
+        doc="Operation cost of lime feed system - energy consumption, labor, and maintenance",
+        domain=NonNegativeReals,
+        units=blk.costing_package.base_currency,
+    )
+
+    blk.soda_ash_add_op_cost = Var(
+        initialize=1e4,
+        doc="Operation cost of adding soda ash - chemical addition cost",
+        domain=NonNegativeReals,
+        units=blk.costing_package.base_currency,
+    )
+
+    if blk.unit_model.config.silica_removal:
+        blk.mg_add_op_cost = Var(
+            initialize=1e4,
+            doc="Operational cost of adding Mg - chemical addition cost",
+            domain=NonNegativeReals,
+            units=blk.costing_package.base_currency,
+        )
+
+    blk.lime_sludge_mngt_op_cost = Var(
+        initialize=1e5,
+        doc="Operational cost of lime sludge management",
+        domain=NonNegativeReals,
+        units=blk.costing_package.base_currency,
+    )
+
+    blk.admin_op_cost = Var(
+        initialize=1e5,
+        doc="Operational cost of administrative, laboratory and maintenance building",
+        domain=NonNegativeReals,
+        units=blk.costing_package.base_currency,
+    )
+
+    # endregion
+
+    # region Capital cost component constraints
+
+    # Mixing tank
+    if value(blk.unit_model.vel_gradient_mix) <= 300:
```

Comment:
I'm not wild about using ``value`` here because it typically evaluates at the initialized value (I think)... but interested to hear @adam-a-a thoughts.

But also- the difference in power demand between each of the velocity gradients can't be that high (correct me if I'm wrong). So maybe we just use the middle one?

Reply:
Oh yeah- since `vel_gradient_mix` is a variable, you can imagine the case where the user unfixes this variable. Now, the value will be changing with every solver iteration. A potential way around this, and I guess this is slightly more advanced and has some known issues in particular circumstances, would be to use an expression if (`Expr_IF`) rather than these if-else conditionals. We can talk more about it, but another approach could be to simplify if possible for a first iteration of this model.

Reply:
Removed value and if statement. The coefficients of the equation in each scenario is now a mutable parameter.